hazelcast/src/main/java/com/hazelcast/instance/Node.java

```diff
@@ -330,11 +330,11 @@ void start() {
                     hazelcastThreadGroup.getThreadNamePrefix("MulticastThread"));
             multicastServiceThread.start();
         }
-        if (groupProperties.getBoolean(GroupProperty.DISCOVERY_SPI_ENABLED)) {
+        if (properties.getBoolean(GroupProperty.DISCOVERY_SPI_ENABLED)) {
```

Comment:
There are over 350 hits on `(GroupProperty.` which matches that pattern. I don't intend to cleanup all of these in this PR.